components/dashboard/src/components/Button.tsx

```diff
@@ -30,7 +30,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         {
             type = "primary",
             className,
-            htmlType,
+            htmlType = "button",
```

Comment:
This was important to have `button` be the default, otherwise they can cause problems in forms inside of a Modal w/o a type set.